core/deployment/src/main/java/io/quarkiverse/langchain4j/deployment/MethodParameterAsTemplateVariableAllowance.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkiverse.langchain4j.deployment;
+
+public enum MethodParameterAsTemplateVariableAllowance {
```

Comment:
I would like some Javadoc about this class

Reply:
I found this idea quite interesting! Nice!